vendor/github.com/vmware/govmomi/govc/device/cdrom/insert.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2014-2015 VMware, Inc. All Rights Reserved.
+Copyright (c) 2014-2016 VMware, Inc. All Rights Reserved.
```

Comment:
This the vendor code, so it's different with our own copyright.
In general, the left number means which year the file is created, and the right number means which year the last change happens.